packages/gatsby-theme-carbon/src/components/InlineNotification/InlineNotification.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { InlineNotification as CarbonInlineNotification } from 'carbon-components-react';
+import { Row, Column } from '../Grid';
+
+import { notification } from './InlineNotification.module.scss';
+
+export default class InlineNotification extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    kind: PropTypes.oneOf(['error', 'info', 'success', 'warning']),
+  };
+
+  static defaultProps = {
+    kind: 'info',
+  };
+
+  render() {
+    const { children, className, kind } = this.props;
```

Comment:
instead of pulling kind out can we spread the props into the component as the last prop? That way we'd handle all the other props/future props on the Carbon component.

Reply:
I can, but not sure we want all those options available here, since its a little different use-case than the Carbon component? We always want the close button hidden for example, and always the lower contrast.